src/SFA.DAS.Apprenticeships.Web.UnitTests/Controllers/ChangeOfStartDate/ChangeOfStartDateEmployerControllerTests.cs

```diff
@@ -0,0 +1,76 @@
+﻿using AutoFixture;
+using FluentAssertions;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Moq;
+using SFA.DAS.Apprenticeships.Domain.Apprenticeships.Api.Responses;
+using SFA.DAS.Apprenticeships.Domain.Interfaces;
+using SFA.DAS.Apprenticeships.Web.Controllers.ChangeOfStartDate;
+using SFA.DAS.Apprenticeships.Web.Models;
+using SFA.DAS.Apprenticeships.Web.Models.ChangeOfStartDate;
+using SFA.DAS.Apprenticeships.Web.UnitTests.TestHelpers;
+using SFA.DAS.Employer.Shared.UI;
+
+namespace SFA.DAS.Apprenticeships.Web.UnitTests.Controllers.ChangeOfStartDate;
+
+
+[TestFixture]
+public class ChangeOfStartDateEmployerControllerTests
+{
+    private Fixture _fixture;
+    private Mock<ILogger<ChangeOfStartDateEmployerController>> _loggerMock;
+    private Mock<IApprenticeshipService> _apprenticeshipServiceMock;
+    private Mock<IMapper> _mapperMock;
+
+    [SetUp]
+    public void Setup()
+    {
+        _fixture = new Fixture();
+        _loggerMock = new Mock<ILogger<ChangeOfStartDateEmployerController>>();
+        _apprenticeshipServiceMock = new Mock<IApprenticeshipService>();
+        _mapperMock = new Mock<IMapper>();
+    }
+
+    [Test]
+    public async Task ViewPendingChangePage_WhenNoPendingChange_ReturnsNotFound()
+    {
+        // Arrange
+        var controller = new ChangeOfStartDateEmployerController(_loggerMock.Object, _apprenticeshipServiceMock.Object, _mapperMock.Object, GetUrlBuilder());
+
+        // Act
+        var result = await controller.ViewPendingChangePage("employerAccountId", "apprenticeshipHashedId");
+
+        // Assert
+        Assert.IsInstanceOf<NotFoundResult>(result);
+    }
+
+    [Test]
+    public async Task ViewPendingChangePage_WhenProviderInitiated_ReturnsApproveView()
+    {
+        // Arrange
```

Comment:
Reduced slightly, but don't want to create too specific a setup as it may only end up being used for one test. If in future tests there is a large overlap in setup then happy to create a parametrized method to generate the setup once its clear what conditions are required.